### PR TITLE
Rework conversion pipeline

### PR DESCRIPTION
**Pull request by greenfork**
The whole conversion pipeline is in a single file `src/generate_bindings.nim` plus some templates in `src/templates/`.

### CODE REVIEW

**Comment by greenfork** [comment]
@planetis-m would you like to review this or do you have any comments?

File: src/generate_bindings.nim [hunk 1]
```diff
@@ -0,0 +1,673 @@
...
+          # In Model
+          ("meshes* {.importc: \"meshes\".}: ptr Mesh",
+           "meshes* {.importc: \"meshes\".}: ptr UncheckedArray[Mesh]"),
+          ("materials* {.importc: \"materials\".}: ptr Material",
+           "materials* {.importc: \"materials\".}: ptr UncheckedArray[Material]"),
+          ("meshMaterial* {.importc: \"meshMaterial\".}: ptr cint",
+           "meshMaterial* {.importc: \"meshMaterial\".}: ptr UncheckedArray[cint]"),
+          # In ModelAnimation
+          ("framePoses* {.importc: \"framePoses\".}: ptr ptr Transform",
+           "framePoses* {.importc: \"framePoses\".}: ptr UncheckedArray[ptr Transform]"),
```
Comment (planetis-m) [inline src/generate_bindings.nim:537]: That's a ptr UncheckedArray[ptr UncheckedArray[Transform]]

**Comment by planetis-m** [comment]
If you want please copy the wrappers and destructors from my PR. Else I could do it later.

File: src/generate_bindings.nim (hunk 1 above)
Comment (greenfork) [reply src/generate_bindings.nim:537]: Good catch!

**Comment by greenfork** [comment]
> If you want please copy the wrappers and destructors from my PR. Else I could do it later.

I will do it in a separate PR shortly. There could be changes to examples and readme additions

**Pull request merged**